lms/templates/fields/field_readonly.underscore

```diff
@@ -1,10 +1,8 @@
-<label class="u-field-title" aria-hidden="true">
-    <%- title %>
-</label>
-<span class="sr" for="u-field-input-<%- id %>"><%- screenReaderTitle %></span>
-<span class="u-field-value">
-    <input id="u-field-input-<%- id %>" aria-describedby="u-field-message-<%- id %>" type="text" name="input" readonly=true value="<%- value %>">
-</span>
+<% if (title) { %>
+<span class="u-field-title" aria-hidden="true"><%- title %></span>
```

Comment:
As far as title vs. screenReaderTitle, it's actually the opposite-- the screenReader title may have more information than the title; and in particular, in some cases there is no title. The span with the title (if present) is marked aria-hidden, and then the span with the screenReaderTitle is class="sr". Doesn't that prevent 2 overlapping titles?

The ability to specify a screenReaderTitle that is different from the title is a common bit of functionality over all of the fields. I would prefer not to change it in this case.